seed-job/template.yaml

```diff
@@ -65,6 +65,7 @@ objects:
 
                     def image_name_with_registry = "${REGISTRY_URL}/${image_name}"
                     def slave_container_name = "jnlp_ccp-pipeline-master"
+                    def base_image_name= "base/image-with:tag"
```

Comment:
this is not needed

Reply:
as in the value of this variable can be initialized as empty string, i.e. `def base_image_name = ""`
also space before `=` operator is missing

Reply:
done